### ISSUE TITLE
RUNTIME ERROR : Divide by zero encountered in remainder

### ISSUE THREAD

**Issue by bt9411** [buriburisuri/speech-to-text-wavenet#77]
****# **how to fix this is error ? and is it possible train data with CPU ?****** 

RuntimeWarning: divide by zero encountered in remainder
  start_step = sess.run(tf.sg_global_step()) % opt.ep_size
I 0529:15:29:11.231:sg_train.py:301] 	Epoch[000:gs=0] - loss = NA
I 0529:15:29:11.232:sg_train.py:301] 	Epoch[001:gs=0] - loss = NA
I 0529:15:29:11.233:sg_train.py:301] 	Epoch[002:gs=0] - loss = NA
I 0529:15:29:11.234:sg_train.py:301] 	Epoch[003:gs=0] - loss = NA
I 0529:15:29:11.235:sg_train.py:301] 	Epoch[004:gs=0] - loss = NA
I 0529:15:29:11.444:sg_train.py:301] 	Epoch[005:gs=0] - loss = NA
I 0529:15:29:11.444:sg_train.py:301] 	Epoch[006:gs=0] - loss = NA
I 0529:15:29:11.445:sg_train.py:301] 	Epoch[007:gs=0] - loss = NA
I 0529:15:29:11.446:sg_train.py:301] 	Epoch[008:gs=0] - loss = NA
I 0529:15:29:11.447:sg_train.py:301] 	Epoch[009:gs=0] - loss = NA
I 0529:15:29:11.448:sg_train.py:301] 	Epoch[010:gs=0] - loss = NA
I 0529:15:29:11.448:sg_train.py:301] 	Epoch[011:gs=0] - loss = NA
I 0529:15:29:11.449:sg_train.py:301] 	Epoch[012:gs=0] - loss = NA
I 0529:15:29:11.449:sg_train.py:301] 	Epoch[013:gs=0] - loss = NA
I 0529:15:29:11.450:sg_train.py:301] 	Epoch[014:gs=0] - loss = NA
I 0529:15:29:11.451:sg_train.py:301] 	Epoch[015:gs=0] - loss = NA
I 0529:15:29:11.451:sg_train.py:301] 	Epoch[016:gs=0] - loss = NA
I 0529:15:29:11.452:sg_train.py:301] 	Epoch[017:gs=0] - loss = NA
I 0529:15:29:11.452:sg_train.py:301] 	Epoch[018:gs=0] - loss = NA
I 0529:15:29:11.453:sg_train.py:301] 	Epoch[019:gs=0] - loss = NA
I 0529:15:29:11.455:sg_train.py:301] 	Epoch[020:gs=0] - loss = NA
I 0529:15:29:11.456:sg_train.py:301] 	Epoch[021:gs=0] - loss = NA
I 0529:15:29:11.457:sg_train.py:301] 	Epoch[022:gs=0] - loss = NA
I 0529:15:29:11.459:sg_train.py:301] 	Epoch[023:gs=0] - loss = NA
I 0529:15:29:11.460:sg_train.py:301] 	Epoch[024:gs=0] - loss = NA
I 0529:15:29:11.461:sg_train.py:301] 	Epoch[025:gs=0] - loss = NA
I 0529:15:29:11.462:sg_train.py:301] 	Epoch[026:gs=0] - loss = NA
I 0529:15:29:11.463:sg_train.py:301] 	Epoch[027:gs=0] - loss = NA
I 0529:15:29:11.464:sg_train.py:301] 	Epoch[028:gs=0] - loss = NA
I 0529:15:29:11.464:sg_train.py:301] 	Epoch[029:gs=0] - loss = NA
I 0529:15:29:11.465:sg_train.py:301] 	Epoch[030:gs=0] - loss = NA
I 0529:15:29:11.466:sg_train.py:301] 	Epoch[031:gs=0] - loss = NA
I 0529:15:29:11.467:sg_train.py:301] 	Epoch[032:gs=0] - loss = NA
I 0529:15:29:11.467:sg_train.py:301] 	Epoch[033:gs=0] - loss = NA
I 0529:15:29:11.468:sg_train.py:301] 	Epoch[034:gs=0] - loss = NA
I 0529:15:29:11.469:sg_train.py:301] 	Epoch[035:gs=0] - loss = NA
I 0529:15:29:11.470:sg_train.py:301] 	Epoch[036:gs=0] - loss = NA
I 0529:15:29:11.472:sg_train.py:301] 	Epoch[037:gs=0] - loss = NA
I 0529:15:29:11.473:sg_train.py:301] 	Epoch[038:gs=0] - loss = NA
I 0529:15:29:11.475:sg_train.py:301] 	Epoch[039:gs=0] - loss = NA
I 0529:15:29:11.476:sg_train.py:301] 	Epoch[040:gs=0] - loss = NA
I 0529:15:29:11.477:sg_train.py:301] 	Epoch[041:gs=0] - loss = NA
I 0529:15:29:11.478:sg_train.py:301] 	Epoch[042:gs=0] - loss = NA
I 0529:15:29:11.478:sg_train.py:301] 	Epoch[043:gs=0] - loss = NA
I 0529:15:29:11.479:sg_train.py:301] 	Epoch[044:gs=0] - loss = NA
I 0529:15:29:11.480:sg_train.py:301] 	Epoch[045:gs=0] - loss = NA
I 0529:15:29:11.480:sg_train.py:301] 	Epoch[046:gs=0] - loss = NA
I 0529:15:29:11.481:sg_train.py:301] 	Epoch[047:gs=0] - loss = NA
I 0529:15:29:11.481:sg_train.py:301] 	Epoch[048:gs=0] - loss = NA
I 0529:15:29:11.482:sg_train.py:301] 	Epoch[049:gs=0] - loss = NA
I 0529:15:29:11.483:sg_train.py:301] 	Epoch[050:gs=0] - loss = NA

**Comment by DontStopMeNow** [comment]
@batbaji Hi! I have the same problem. Have you fixed it?

**Comment by imarvinds** [comment]
I have the same issue. Please help us fix it.

**Comment by profsatwinder** [comment]
same issue!
